usagov/vote-gov-drupal
# VOTE-904 add languages to CMS

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-904

## Description (optional)
Added all languages to CMS

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. run `lando drush cim`

### Testing steps
1. Go to http://vote-gov-d10.lndo.site/admin/config/regional/language
2. Make sure all the languages of the current site are accounted for
